templates/tydiqa/primary_task/templates.yaml

```diff
@@ -0,0 +1,156 @@
+dataset: tydiqa
+subset: primary_task
+templates:
```

Comment:
**document_plaintext**: Document plaintext is large and may not be feed into the neural network in a single sample. Ideally, the `document_plaintext` should be divided into multiple passages by `passage_answer_candidates.start_byte` and `passage_answer_candidates.end_byte`. Out of multiple passages, we may find the answer. The total number of passages may be >= 100 for a sample. So I thought it doesn't make sense to add the `document_plaintext` since it is not a "YES/NO" question. Rather it asks a simple question that if the Answer is "YES/NO". I believe it should be solvable without passage in large LM.
**task_template** : This sounds wired but this is their primary task. Reading comprehension was their secondary task. See here, https://github.com/google-research-datasets/tydiqa
So when I filled up the template for the primary task, I keep them as `task_template=True`.

Reply:
hmmm i'd rather separate the issue of prompting for the task and the tokenization/model separately. ultimately truncating will be handled on the model/tokenization side so you shouldn't worry too much about what's long or not.

ok indeed, yes/no is part of the intended task. though span extraction reading comprehension is also one of the primary task
<img width="881" alt="Capture d’écran 2021-06-15 à 4 42 20 PM" src="https://user-images.githubusercontent.com/16107619/122120712-c07e0580-cdf8-11eb-8d25-6a796b0f24b5.png">

Please add one or two templates using  `document_plaintext`

Reply:
Can you check now?